workers/scanner/src/main/kotlin/scanner/OrtServerScanResultStorage.kt

```diff
@@ -85,6 +89,12 @@ class OrtServerScanResultStorage(
     private val db: Database,
     private val scannerRunId: Long
 ) : ProvenanceBasedScanStorage {
+    /**
+     * A [Map] to store the issues encountered for the scanned provenances. This is used to track issues that need to
+     * be associated with the current ORT run.
+     */
+    private val issuesMap = ConcurrentHashMap<Provenance, Set<Issue>>()
```

Comment:
What is the lifespan for this collection ?

Is the `OrtserverScanResultStorage` instantiated and destroyed per ORT run or is it per storage service i.e. persistent in the application ?

Reply:
The lifespan is the current ORT run only. For each run, a dedicated instance of `OrtServerScanResultStorage` is created.